css/main.css

```
@@ -417,15 +417,53 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
-.not-active {
+.invisible {
   display: none;
 }
 
 #close {
   align-self: flex-end;
 }
 
+#mobile-nav {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 100;
+  background-color: #3c3a39;
+  width: 375px;
+  height: 100vh;
+}
+
+#mobile-nav a {
+  color: #fff5e1;
+}
+
+#mobile-nav li {
+  border-bottom: #fff5e1 solid 1px;
+  margin-top: 5%;
+  text-align: start;
+}
+
+#mobile-nav div {
+  display: flex;
+  flex-direction: column;
+  margin: 10% auto;
+  width: 80%;
+  height: 30%;
+}
+
+#mobile-nav ul {
+  display: flex;
+  flex-direction: column;
+}
+
```

- [ ] Please implement the changes asked by the previous reviewer, those points are valid points and will help you to re-use your style without *inventing a new wheel*.
- [ ] Please follow the `DRY` rule, your code is repeating and it will make your code lengthier and hard to `maintain`. Kindly consider the below example.
```suggestion
#mobile-nav div, #mobile-nav ul {
  display: flex;
  flex-direction: column;
}

#mobile-nav div {
  margin: 10% auto;
  width: 80%;
  height: 30%;
}
```

- `[OPTIONAL]` Please try to avoid the use of height, dealing with the height in `css` make sometimes really difficult to make our design responsive. I would really like to suggest you always let the child components decide the height of the container. which is `div` in your code snippet. with the help of `padding` you can easily adjust the height of container.